backend/app/core/logging.py

```diff
@@ -0,0 +1,33 @@
+
+import logging
+from pythonjsonlogger.json import JsonFormatter
+from .config import Settings
+
+logger = logging.getLogger(__name__)
+
+def setup_logging(settings: Settings):
```

Comment:
I like this implementation a lot. One question for us is whether we'd prefer to offload this configuration to the `.env` or similar env-specific config file instead of in the code